Backend/dev/test-tool/context-api/server.py

```
@@ -336,6 +342,221 @@ def get_full_context():
+            "voidedInvoices": voided_inv,
+        },
+    }
+
+
+def clear_finance_data(schema):
+    """Delete all finance data (ledger entries, invoices, links, accounts) for a clean slate."""
+    try:
+        conn = get_conn()
+        conn.autocommit = True
+        cur = conn.cursor()
+        cur.execute(f"DELETE FROM {schema}.finance_invoice_ledger_link")
```

Ruff (0.15.10)

[error] Possible SQL injection vector through string-based query construction

(S608)